pymc3/tests/test_examples.py

```diff
@@ -16,13 +16,16 @@ def check_example(example_name):
 
 
```

Comment:
Yup. Feel free to do something more elegant here. I just wanted to get things working again.